x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/CommonEqlActionTestCase.java

```diff
@@ -146,7 +146,7 @@ protected EqlSearchResponse runQuery(String index, String query, boolean isCaseS
         request.tiebreakerField("event.sequence");
         // some queries return more than 10 results
         request.size(50);
-        request.fetchSize(2);
+        request.fetchSize(randomIntBetween(2, 50));
```

Comment:
@matriv see this one as well. It unraveled a bug - when the fetchSize was 5, the results were different due to us skipping a window.